homework/Operators.js

```diff
@@ -0,0 +1,46 @@
+/* I DO NOT FEEL COMFORTABLE DOING THIS ASSIGNMENT AS I NEED TO STUDY MORE. SUBMITTING IT AS IT IS 
+BECAUSE I AM ALREADY LATE*/
+
+/* Consider the following snippet:
+
+0 == '0'; // true
+0 == [];  // true
+"0" == [];  // false
+Mathematically, if a == b and b == c, then a == c. 
+Can you explain in your words why this is not true for the above snippet? */
+
+var a = "0";
+var b = 0;
+var c = [];
+
+console.log(a == b ? "true" : "false");
+console.log(b == c ? "true" : "false");
+console.log(a == c ? "true" : "false");
+// 1. '"a" is a string while "c" is an Array';
+
```

Comment:
Very good... For more info look at this article [https://www.freecodecamp.org/news/explaining-the-best-javascript-meme-i-have-ever-seen/](url) (5 minute read)